bugbug/models/__init__.py

```diff
@@ -1 +1,28 @@
 # -*- coding: utf-8 -*-
+import logging
+import os
+
+from bugbug.model import BugModel
+from bugbug.models.component import ComponentModel
+from bugbug.models.defect_enhancement_task import DefectEnhancementTaskModel
+from bugbug.models.regression import RegressionModel
+
+LOGGER = logging.getLogger()
+
+
+MODELS = {
+    "defectenhancementtask": DefectEnhancementTaskModel,
+    "component": ComponentModel,
+    "regression": RegressionModel,
+    "bug": BugModel,
+}
+
+
+def load_model(model_name, model_dir=None):
+    if model_dir is None:
+        model_dir = "."
+
```

Comment:
Could you add an assertion that `model_name` is in MODELS?

The code would fail anyway in line 27 if that's not the case, but it'd be nice to have a better error message.